### PR TITLE
raw values type

### PR DESCRIPTION
**Pull request by nedmsmith**
Added text to the Raw Values Type section. Fixes issue #9, partially

### CODE REVIEW

File: draft-ietf-rats-corim.md [hunk 1]
```diff
@@ -991,7 +991,19 @@ computing base.
...
 ###### Raw Values Types {#sec-comid-raw-value-types}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/9
+Raw value measurements are typically vendor defined values that are checked by Verifiers
+for consistency only, since the security relevance is opaque to Verifiers.
+
+There are two parts to a `raw-value-group`, a measurement and an optional mask.
+The default raw value measurement is a CBOR tagged `bstr`.
+Additional raw value types can be defined, but must be CBOR tagged so that parsers can distinguish
+between the various forms.
```
[the remark this answers is not in the document]
Comment (henkbirkholz) [reply draft-ietf-rats-corim.md:1000]: This is a nit (but "forms"... 🙃)

File: draft-ietf-rats-corim.md [hunk 2]
```diff
@@ -991,7 +991,19 @@ computing base.
 
 ###### Raw Values Types {#sec-comid-raw-value-types}
 
-[^issue] https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/9
+Raw value measurements are typically vendor defined values that are checked by Verifiers
+for consistency only, since the security relevance is opaque to Verifiers.
+
+There are two parts to a `raw-value-group`, a measurement and an optional mask.
```
Comment (henkbirkholz) [inline draft-ietf-rats-corim.md:997]: > optional mask

Just looking at this PR and not the whole document: is it clear what mask means here?

File: draft-ietf-rats-corim.md [hunk 3]
```diff
@@ -991,7 +991,19 @@ computing base.
...
+The default raw value measurement is a CBOR tagged `bstr`.
+Additional raw value types can be defined, but must be CBOR tagged so that parsers can distinguish
+between the various forms.
+
+The mask is applied by the Verifier as part of appraisal.
+Only the raw value bits with corresponding TRUE mask bits are compared during appraisal.
```
Comment (henkbirkholz) [inline draft-ietf-rats-corim.md:1003]: See above. Does this text lead to implementable behavior?

File: draft-ietf-rats-corim.md (hunk 2 above)
Comment (yogeshbdeshpande) [reply draft-ietf-rats-corim.md:997]: issue #9 will be worked further

**Review by andrew-draper — APPROVED** [review]
Text in line 1003 could be better phrased, but this change is better than the previous document so approving

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM!

**Pull request merged**